suites/nautilus/rbd/rbd_mirror_using_ceph_ansible.yaml

```diff
@@ -0,0 +1,116 @@
+#===============================================================================================
+# Tier-level: 1
+# Test-Scenario: Configure RBD Mirror using ceph-ansible and run IOs
+#
+# Cluster Configuration:
+#    conf/nautilus/rbd/rbd_mirror_using_ceph_ansible.yaml
+#    No of Clusters : 2
+#    Each cluster configuration
+#    6-Node cluster(RHEL-8.3 and above)
+#    3 MONS, 2 MGR, 3 OSD and 1 RBD MIRROR nodes
+#     Node1 - Mon, Mgr, Installer
+#     Node2 - client, rbd-mirror
+#     Node3 - Mon, OSD, mgr
+#     Node4 - Mon, OSD
+#     Node5 - OSD
+#
+# Sequence of tests -
+#   1) Configure two clusters
+#   2) Handle same cluster names (required for ceph-ansible usage)
+#   3) Add rbd mirror daemon to both clusters
+#   4) Run IOs
+#===============================================================================================
+tests:
+  - test:
+      name: install ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: True
+
+  - test:
+      name: ceph ansible
+      module: test_ansible.py
+      clusters:
+        ceph-rbd1:
+          config:
+            ansi_config:
+              ceph_origin: distro
+              osd_auto_discovery: False
+              ceph_stable_rh_storage: True
+              fetch_directory: ~/fetch
+              copy_admin_key: true
+              dashboard_enabled: False
+              osd_scenario: lvm
+              ceph_conf_overrides:
+                global:
+                  osd_pool_default_pg_num: 64
+                  osd_pool_default_pgp_num: 64
+                  mon_max_pg_per_osd: 1024
+                mon:
+                  mon_allow_pool_delete: true
+        ceph-rbd2:
+          config:
+            ansi_config:
+              ceph_origin: distro
+              osd_auto_discovery: False
+              fetch_directory: ~/fetch
+              copy_admin_key: true
+              dashboard_enabled: False
+              osd_scenario: lvm
+              ceph_conf_overrides:
+                global:
+                  osd_pool_default_pg_num: 64
+                  osd_pool_default_pgp_num: 64
+                  mon_max_pg_per_osd: 1024
+                mon:
+                  mon_allow_pool_delete: true
+      desc: setup dual cluster using ceph-ansible
+      destroy-cluster: false
+      abort-on-fail: true
+
+  - test:
+      name: rbd mirror pre-req
+      module: mirror_setup.py
+      clusters:
+        ceph-rbd1:
+          config:
+            way: two-way
+            used_ceph-ansible: True
+      desc: pre-req for mirroring host
+      abort-on-fail: true
+
+  - test:
+      name: Add rbd-mirror daemons
+      module: configure_rbd_mirror.py
```

Comment:
Do we need a separate module? As we are already adding `[rbdmirrors]` section in the inventory file and are executing the playbooks in the above tests.

Reply:
We need pools to be created before initiating plabook to configure mirror daemon
In first initiation though its added to inventory, since ceph_rbd_mirror_configure is not set to tru, mirroring daemon would not be created.
So this module is to add configs to rbdmirrors.yaml and to initiate playbook limiting to rbdmirror daemons.

Reply:
@psathyan Can you please let me know if we can mark conversation as resolved?